definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2711,19 +2711,6 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "01/01/2019",
-    "CaseTypeID": "DIVORCE",
-    "ID": "allowPetitionerToAmend",
-    "Name": "Allow Petitioner to amend",
-    "Description": "Allow Petitioner to amend",
-    "DisplayOrder": 23,
-    "PreConditionState(s)": "AwaitingHWFDecision;Submitted;Issued;Rejected;PendingRejection;solicitorAwaitingPaymentConfirmation;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService",
-    "PostConditionState": "AwaitingAmendCase",
```

Comment:
I'm not sure why this would raise the error Gurdeep shared as this state is defined in State.json.
Did it solve it?

Reply:
It's not, actually. It's in the nonprod version.

Reply:
I mean. The State is not in the State.json file.